templates/attend-status.html.tpl

```diff
@@ -0,0 +1,21 @@
+<div id="attend" {{ if .oob }}hx-swap-oob="true" {{ end }}>
```

Comment:
Why do use `oob` instead of putting `hx-target="#attend` on the buttons? Maybe im not oobular enough but I don't know what the difrene is

Reply:
Good point, will change